From 3229ca31e59933608f82001c1cdcca9d0a0aa0e0 Mon Sep 17 00:00:00 2001
From: Anton Antonov <Anton.Antonov@arm.com>
Date: Wed, 31 Aug 2022 17:19:08 +0100
Subject: [PATCH] Subject: [PATCH] Pass PSA_TARGET_QCBOR to psa-arch-tests

psa-arch-tests require they own version of qcbor library.
Pass PSA_TARGET_QCBOR which defines where pre-fetched qcbor sources are.

Upstream-Status: Pending
Signed-off-by: Anton Antonov <Anton.Antonov@arm.com>
---
 external/psa_arch_tests/pas-arch-test-init-cache.cmake.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/external/psa_arch_tests/pas-arch-test-init-cache.cmake.in b/external/psa_arch_tests/pas-arch-test-init-cache.cmake.in
index 5c63596..64196c2 100644
--- a/external/psa_arch_tests/pas-arch-test-init-cache.cmake.in
+++ b/external/psa_arch_tests/pas-arch-test-init-cache.cmake.in
@@ -10,6 +10,7 @@ set(CMAKE_TOOLCHAIN_FILE "@TS_EXTERNAL_LIB_TOOLCHAIN_FILE@" CACHE STRING "")
 
 set(TOOLCHAIN INHERIT CACHE STRING "")
 set(PSA_INCLUDE_PATHS "@PSA_ARCH_TESTS_EXTERNAL_INCLUDE_PATHS@"  CACHE STRING "")
+set(PSA_TARGET_QCBOR "@PSA_TARGET_QCBOR@"  CACHE STRING "")
 set(SUITE "@TS_ARCH_TEST_SUITE@"  CACHE STRING "")
 set(ARCH_TEST_EXTERNAL_DEFS "@PSA_ARCH_TEST_EXTERNAL_DEFS@"  CACHE STRING "")
 set(CMAKE_VERBOSE_MAKEFILE OFF CACHE BOOL "")
-- 
2.25.1

